Multisig/UI/Transactions/TransactionDetails/TransactionStatusTypeView.swift

```diff
@@ -41,16 +40,15 @@ struct TransactionStatusTypeView: View {
     var imageName: String {
         if let _ = transaction as? CustomTransaction {
             return "ico-custom-tx"
-        }
-        else if let transfer = transaction as? TransferTransaction {
+        } else if let transfer = transaction as? TransferTransaction {
             return transfer.isOutgoing ? "ico-outgoing-tx" : "ico-incoming-tx"
         }
 
         return "ico-settings-tx"
     }
 
     var displayStatusOnSecondLine: Bool {
-        if [TransactionStatus.waitingConfirmation, TransactionStatus.waitingExecution].contains(transaction.status) {
+        if [.waitingConfirmation, .waitingExecution].contains(transaction.status) {
```

Comment:
the condition is repeated in multiple places, time to extract it in the computed variable of the TransactionStatus